.github/workflows/build.yml

```diff
@@ -2,24 +2,100 @@ name: Build
 
 on:
   workflow_call:
-    secrets:
-      BITBUCKET_READ_ONLY_SSH_KEY:
+    inputs:
```

Comment:
This should be in a separate workflow, `build-larva.yml` or something similarly named. Most, but not all, of our themes use Larva, so we'll need to include this step on a per-theme basis. The `build.yml` job is a placeholder for when we move CSS and JS builds into pipelines, rather than committing those built assets as we currently do.

Reply:
i've updated this as recommended and moved it to `build-larva.yml`